lib/workers/repository/update/branch/auto-replace.spec.ts

```diff
@@ -30,6 +30,7 @@ describe('workers/repository/update/branch/auto-replace', () => {
       upgrade = getConfig() as BranchUpgradeConfig;
       upgrade.packageFile = 'test';
       upgrade.manager = 'html';
+      upgrade.autoReplaceRegexGlobalMatch = true;
```

Comment:
I think the `getConfig()` above will already set the default value.


```suggestion
```

Reply:
I was also expecting the same thing, but unfortunately it did not.  When debugging i noticed that the value was `undefined`.

Reply:
works after removing parent=packageRule from configuration.  thanks viceice